internal/cfg_test.go

```diff
@@ -67,7 +69,7 @@ func TestReadCfg(t *testing.T) {
 
 		_ = os.Setenv(`SCM_WORKSPACE_DIR`, `~/Workspace`)
 		expected := Cfg{
-			ScmPostCloneCmd: NewPostCloneCmd("idea", []string{"/Users/pkorobeinikov/Workspace/github.com/user/repo"}),
```

Comment:
👍

Reply:
So, what about to use `path.Join()` instead of `+` operator?

Reply:
I think filepath.Join  would be better. 
Done